app/views/christmas_money_planner/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<% set_meta_tags(title:       t('christmas_money_planner.meta.title'),
+                 description: t('christmas_money_planner.meta.description'),
+                 canonical:   t('christmas_money_planner.meta.canonical'),
+                 alternate:   t('christmas_money_planner.meta.alternate')) %>
+
+<% content_for(:context_bar) do %>
+  <%= render 'shared/breadcrumbs', breadcrumbs: breadcrumbs %>
+<% end %>
+
+<% content_for(:alternate_link) do %>
+  <%= link_to(alternate_url, lang: alternate_locale, class: "t-#{alternate_locale}-link") do %>
+    <span class="icon icon--globe"></span>
+    <%= t("locales.#{alternate_locale}") %>
+  <% end %>
+<% end %>
+
+<div class="l-constrained">
+  <div class="l-container-tool">
+    <!--<h1><%= t('christmas_money_planner.page_heading') %></h1>
```

Comment:
Good catch, they have now been removed and their corresponding entries in en & cy .yml